src/main/java/core/basesyntax/app/Main.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.app;
+
+import core.basesyntax.service.OperationHandler;
+import core.basesyntax.service.OperationStrategy;
+import core.basesyntax.service.Reader;
+import core.basesyntax.service.ReportGenerator;
+import core.basesyntax.service.TransactionParser;
+import core.basesyntax.service.Writer;
+import core.basesyntax.service.impl.CsvReader;
+import core.basesyntax.service.impl.CsvWriter;
+import core.basesyntax.service.impl.ReportGeneratorImpl;
+import core.basesyntax.service.impl.TransactionProcessor;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import core.basesyntax.strategy.handler.BalanceHandler;
+import core.basesyntax.strategy.handler.PurchaseHandler;
+import core.basesyntax.strategy.handler.ReturnHandler;
+import core.basesyntax.strategy.handler.SupplyHandler;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        final String inputFilePath = "src/main/resources/input.csv";
+        final String outputFilePath = "src/main/resources/output.csv";
+        Reader reader = new CsvReader();
```

Comment:
According to the checklist, avoid hardcoding file paths in your solution. It's better to pass them as arguments to the `main()` method or read them from a configuration file or environment variable.